enterprise/dev/ci/internal/ci/changed/diff.go

```diff
@@ -104,6 +104,12 @@ func ParseDiff(files []string) (diff Diff) {
 		if strings.HasPrefix(p, "doc/") && p != "CHANGELOG.md" {
 			diff |= Docs
 		}
+		if strings.HasSuffix(p, ".yaml") || strings.HasSuffix(p, ".yml") {
+			diff |= Docs
+		}
+		if strings.HasSuffix(p, ".json") || strings.HasSuffix(p, ".jsonc") {
```

Comment:
Nit: I've seen `.json5` files in the past (e.g. Renovate configs)

```suggestion
		if strings.HasSuffix(p, ".json") || strings.HasSuffix(p, ".jsonc") || strings.HasSuffix(p, ".json5") {
```